_data/domains.yml

```
@@ -461,7 +461,8 @@ websites:
       img: porkbun.png
       tfa: Yes
       software: Yes
-      doc: https://porkbun.com/products/two_factor_authentication
+      hardware: Yes
+      doc: https://kb.porkbun.com/category/20-accounts
```

I think article 19 fits better as it describes how to enable 2FA.
```suggestion
      doc: https://kb.porkbun.com/article/19-how-to-enable-two-factor-authentication
```